packages/useful-types/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../tsconfig_base.json",
+  "compileOnSave": false,
+  "compilerOptions": {
+    "outDir": "./dist",
+    "baseUrl": "./src",
+    "rootDir": "./src"
+  },
+  "include": [
+    "../../config/typescript/**/*",
```

Comment:
What is this pulling in?

Reply:
Doesn't seem like anything. I wrote a script that converted all `tsconfig.build.json
` => `tsconfig.json` and updated the extended field. Looking at the history of this file this must have been included from the generators.